src/System.Private.CoreLib/src/System/Delegate.cs

```diff
@@ -463,25 +464,25 @@ public static Delegate CreateDelegate(Type type, MethodInfo method, bool throwOn
         }
 
         // V2 API.
-        public static Delegate CreateDelegate(Type type, object firstArgument, MethodInfo method)
+        public static Delegate? CreateDelegate(Type type, object? firstArgument, MethodInfo method)
```

Comment:
I'll stop commenting on these, but please review the Delegate return types of all these methods; many of them should not be nullable I believe.